libraries/resource_hab_package.rb

```diff
@@ -24,7 +24,7 @@ class HartPackage < Chef::Resource::Package
       resource_name :hab_package
       provides :hab_package
 
-      property :bldr_url, String, default: 'https://bldr.habitat.sh'
```

Comment:
This would be removed with edit suggestion to `libraries/provider_hab_package.rb`

Reply:
if we remove that default, then the `/libraries/resource_hab_sup.rb` would need to be updated to specify the public builder by default I think, currently a lot of installs use the public builder for pulling down hab-sup + hab-launcher by default, but then private for everything if the default changes to ""

Reply:
at least I think that's what's going on 😕